launcher/downloader/__init__.py

```diff
@@ -11,3 +13,23 @@ def get_handler_for_url(url: str) -> Base:
         'github.com': Github,
         'www.moddb.com': ModDB,
     }.get(host)(url)
+
+
+@retry(stop=stop_after_attempt(3))
+def _download_mod(url: str, download_dir: Path, use_cached: bool = True) -> Path:
```

Comment:
Drop the first `_` since this is no longer a protected method of FullInstall class